source/installation/install-kafka/kafka-intro.txt

```diff
@@ -0,0 +1,78 @@
+.. _kafka-intro:
+
+==============================================
+Install Relational Migrator with Kafka Connect
+==============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use Relational Migrator to migrate data to and from Kafka. 
+Relational Migrator can be used as a Kafka Connect plug-in that improves 
+resilience and scalability of big data migration jobs.
+
+Use Cases
+---------
+
+Relational Migrator with Kafka is intended for large migration jobs that normally
+require long run times. By using Relational Migrator with Kafka, you help reduce 
+migration times. 
+
+Behavior
+--------
+
+Relational Migrator works with Kafka as a Kafka Connect sink connector. A 
+`Debezium connector <https://debezium.io/documentation/reference/stable/connectors/index.html>`__ 
+extracts data from a SQL database and is an event source. Relational Migrator 
+interprets and transforms events from the Debezium source and sends them to a 
+MongoDB cluster which acts as a data sink.
+
+Relational Migrator is not packaged with Kafka Connect. To use Relational Migrator 
+with Kafka you have to install Relational Manager and integrate it with your 
```

Comment:
([imperative mood](https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/))
In this case it's probably fine to just say something like:
"To use Relational Migrator with Kafka, install Relational ..."

Reply:
Done.